content/java/concepts/array-list/terms/indexOf/indexOf.md

```diff
@@ -0,0 +1,92 @@
+---
+Title: '.indexOf()'
+Description: 'If ArrayList contains given element returns the index of the first occurrence of it , else -1'
```

Comment:
```suggestion
Description: 'Returns the index of the first occurrence of a given element, or -1 if not found.'
```

Reply:
Since this is a term entry, the `Description` should begin with a present-tense verb.

Reply:
ok